stock_picking_line_sequence/models/stock.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    # re-defines the field to change the default
+    sequence = fields.Integer('HiddenSequence',
+                              default=9999)
+
+    # displays sequence on the stock moves
+    sequence2 = fields.Integer('Sequence',
+                               help="Shows the sequence in the Stock Move.",
+                               related='sequence', readonly=True, store=True)
+
+    @api.model
+    def create(self, values):
+        move = super(StockMove, self).create(values)
+        # We do not reset the sequence if we are copying a complete picking
+        if self.env.context.get('keep_line_sequence'):
```

Comment:
Here you are doing the complete opposite thing of the comment above. If keep_line_sequence is true you are resetting the sequence.